### PR TITLE
fix: studio unit types in seed data

### PR DESCRIPTION
**Pull request by jaredcwhite**
This PR addresses #691

- [ ] Addresses the issue in full
- [x] Addresses only certain aspects of the issue

## Description

Re: the issue, there seems to be a data problem (and possibly a bug causing that data issue) in production re: studio/SRO units and numBedrooms, but that isn't addressed here. This PR simply addresses seed data for local dev/testing.

## How Can This Be Tested/Reviewed?

Run the DB seed, and then check that listing filtering works as expected.

## Author Checklist:

- [ ] Added QA notes to the issue with applicable URLs
- [ ] Reviewed in a desktop view
- [ ] Reviewed in a mobile view
- [ ] Reviewed considering accessibility
- [x] Added tests covering the changes
- [ ] Made corresponding changes to the documentation
- [ ] Ran `yarn generate:client` and/or created a migration when required

## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates

### CODE REVIEW

File: api/prisma/seed-staging.ts [hunk 1]
```diff
@@ -972,13 +972,13 @@ export const stagingSeed = async (
...
           numBathrooms: 1,
-          numBedrooms: 1,
+          numBedrooms: 0,
           number: '101',
           sqFeet: '750.00',
           amiChart: { connect: { id: amiChart.id } },
           unitTypes: {
             connect: {
-              id: unitTypes[1].id,
+              id: unitTypes[5].id,
```
Comment (emilyjablonski) [inline api/prisma/seed-staging.ts:981]: Curious why this is 5 and not 0 when it is 0 above when numBedrooms is 0?

Comment (mcgarrye) [reply api/prisma/seed-staging.ts:981]: SRO is different than Studio

**Comment by emilyjablonski** [comment]
@mcgarrye https://github.com/bloom-housing/bloom/pull/4185

**Review by mcgarrye — APPROVED** [review]
The filter works 😭 thank goodness LGTM

File: api/prisma/seed-staging.ts (hunk 1 above)
Comment (jaredcwhite) [reply api/prisma/seed-staging.ts:981]: Yeah, the order of IDs in the enum is sort of weird. SRO is after several of the multi-bedroom types.

**Pull request merged**